app/web/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -136,31 +133,54 @@ export default function CreateGroupChat({ className }: { className?: string }) {
                 className={classes.field}
               />
             )}
```

Comment:
I find that this create chat dialog is now always starting with an error message even when I've just opened it! Probably caused by when `to` doesn't exist in the URL param...
![Screenshot 2022-01-05 002324](https://user-images.githubusercontent.com/13669362/148141615-a9694916-56af-4e19-ad8a-515f3eb0edb1.png)